### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by enzoliuyang94**
Added unit tests for simulation.jl and validation.jl

### CODE REVIEW

File: test/simulation.jl [hunk 1]
```diff
@@ -0,0 +1,197 @@
+import Gurobi
+import JSON
+import JuMP
+using WaterModels
+using PyCall
+
```
Comment (jjstickel) [inline test/simulation.jl:6]: Any modules that are needed should be imported (not used) in `runtests.jl` not in the sub-test files.  Shortcuts for WaterModels and InfrastructureModels are already defined there.

File: test/validation.jl [hunk 2]
```diff
@@ -0,0 +1,200 @@
+import Gurobi
+import JSON
+import JuMP
+using WaterModels
+const _WM = WaterModels
+using PyCall
+
```
Comment (jjstickel) [inline test/validation.jl:7]: Any modules that are needed should be imported (not used) in `runtests.jl` not in the sub-test files.  Shortcuts for WaterModels and InfrastructureModels are already defined there.

**Review by jjstickel — CHANGES_REQUESTED** [review]
Please also add "include" lines for the new test files you created in `test/runtests.jl`.

**Comment by jjstickel** [comment]
@enzoliuyang94 , I refactored your tests a bit. Please take a look and let me know if all looks Ok.
@tasseff , @claytonpbarrows , you may wish to review this large set of new tests for WNTR simulation and validation.

**Review by jjstickel — APPROVED** [review]

**Comment by enzoliuyang94** [comment]
> @enzoliuyang94 , I refactored your tests a bit. Please take a look and let me know if all looks Ok.
> @tasseff , @claytonpbarrows , you may wish to review this large set of new tests for WNTR simulation and validation.

Hi Jonathan, these codes look good to me. Thanks!

**Pull request merged**